kobo/apps/stripe/tests/test_organization_usage.py

```diff
@@ -138,7 +138,7 @@ def test_doesnt_include_org_users_with_invalid_plan(self):
             self.expected_file_size() * self.expected_submissions_single
         )
 
-    @pytest.mark.xfail
+    @pytest.mark.performance
```

Comment:
Now that the test is excluded by default, would it be worth cranking up the number of model instances to a number that you would want to use when you run performance tests? (Or, if not, adding a comment describing how someone might use the test to test performance meaningfully?)